ios/SwagLabsMobileApp/Images.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -3,49 +3,49 @@
     {
       "size" : "20x20",
       "idiom" : "iphone",
-      "filename" : "Login_Bot_graphic@2x-white-20@2x.png",
+      "filename" : "Icon_Art (Edit Me)@2x-20@2x.png",
```

Comment:
Do we want to remove the (Edit Me) from these filenames?

Reply:
Fixed it, thanks. I also removed some images that were not needed